packages/time-picker/src/vaadin-time-picker.js

```diff
@@ -421,6 +434,25 @@ class TimePicker extends PatternMixin(InputControlMixin(ThemableMixin(ElementMix
     );
   }
 
+  /**
+   * @param {boolean} focused
+   * @override
+   * @protected
+   */
+  _setFocused(focused) {
+    super._setFocused(focused);
+
+    if (!focused) {
+      this.__commitValueChange();
+
+      // Do not validate when focusout is caused by document
+      // losing focus, which happens on browser tab switch.
+      if (document.hasFocus()) {
```

Comment:
**note:** There can be duplicate validation on `blur` when it coincides with `change`. I considered it not to be a serious issue since (1) the Flow components will no longer rely on client validation timing (2) `validate()` calls themselves are pretty cheap in web-components.